src/Translations.js

```diff
@@ -25,13 +32,25 @@ const ENGLISH_TRANSLATION = {
         "en": "English",
         "nl": "Nederlands"
     },
+    "input_modes": {
+        "switch_to_interactive": "Switch to interactive mode",
+        "switch_to_batch": "Switch to batch input"
+    },
+    "enter": "Enter"
 };
 
 const DUTCH_TRANSLATION = {
     "Papyros": "Papyros",
-    "enter_code": "Voer hier je code in",
-    "enter_input": "Geef hier de invoer voor je code in",
-    "code_output": "Resultaat van de code",
+    "code": "Code",
+    "code_placeholder": "Voer hier je code in",
```

Comment:
```suggestion
    "code_placeholder": "Schrijf hier je code en klik op 'voer uit' om uit te voeren...",
```
Still not totally happy with this phrasing but can't come up with a synonym for "uitvoeren"

Reply:
One option I see is naming the two buttons `Run` and `Stop` (same in Dutch). Then the sentence makes more sense in Dutch. We could also add the typical triangle and square icons to the run and stop buttons, as an extra visual aid.

Maybe also drop the word "here" for consistency with the input field. The word "je" may also be dropped.